package-lock.json

```diff
@@ -6052,21 +5976,13 @@
       }
     },
     "jitsi-meet-electron-utils": {
-      "version": "github:jitsi/jitsi-meet-electron-utils#3b6562f4d6104bfb5099414d1a79d4164767ea9b",
```

Comment:
I've just created a jitsi-meet-electron-utils release. Could you please rerun npm install?